app/Filament/Clusters/ProductIntegrations.php

```diff
@@ -44,5 +44,5 @@ class ProductIntegrations extends Cluster
 
     protected static ?string $navigationGroup = 'Global Administration';
 
-    protected static ?int $navigationSort = 2;
+    protected static ?int $navigationSort = 1;
```

Comment:
Based on the ticket screenshot I believe the intention was for Product Integrations to come last in the 1st level navigation and for all the other items to come before it in the same order as they were in the 2nd level navigation.

Reply:
Sorry for this !
Done. moved Product Integrations at last.